### ISSUE TITLE
[FR] Alter "Shelly Night mode"

### ISSUE THREAD

**Issue by frosenlind** [StyraHem/ShellyForHASS#453]

**Comment by hakana** [comment]
Can you please let me know what you need this for?

**Comment by frosenlind** [comment]
> Can you please let me know what you need this for?

You want the lights to start at a specific brightness value during a certain period of time. 
For example during the night you don't want the bathroom lights to turn on to last known value (witch is often 100%) but to (lets say) 25%.

Today you could do this two ways.
1. By editing night mode in the shelly app.
2. By having an automation listening to "turn on" event on the shelly. 

The problem with nr 1 is having to use the shelly app and not HA.
The problem with nr 2 is that the shelly starts fading up to last known value before the automation kicks in.

Also.
If it would be possible to alter the night mode on/off and the time of the night mode with shelly4hass you would also gain the ability to, with some small automations, set the brightness levels in the house for different times of the day without the need to call HA service turn_on.

**Comment by hakana** [comment]
Will a service call to set the dim level without turning on the light solves the problem?

**Comment by frosenlind** [comment]
In combination with other automations yes.

**Comment by hakana** [comment]
Yes I mean so you can change the dimmer level from automation without turning on the light.

**Comment by frosenlind** [comment]
Yes.

**Comment by Ysbrand** [comment]
Hi,

Yes, adjusting the dimmer without turning on the light would be perfect:
http://<IP>/light/0?brightness=5
This would allow to change the dimming level using an automation. This is helps during nighttime as the night mode is not very flexible.
